.eslintrc

```diff
@@ -1,11 +1,37 @@
-{
-    "extends": [
+{  
+  "extends": [
       "oclif",
       "oclif-typescript"
     ],
     "rules": {
       "unicorn/no-abusive-eslint-disable": "off",
       "@typescript-eslint/no-use-before-define": "off",
-      "@typescript-eslint/ban-ts-ignore": "off"
+      "@typescript-eslint/ban-ts-ignore": "off",
+      "object-curly-spacing": "off",
+      "no-useless-constructor": "off",
+      "@typescript-eslint/no-unused-vars": [
+        "error",
+        {
+            "args": "none"
+        }
+    ],
+    "@typescript-eslint/prefer-namespace-keyword": "error",
+    "@typescript-eslint/quotes": [
+        "error",
+        "single",
+        {
+            "avoidEscape": true,
+            "allowTemplateLiterals": true
+        }
+    ],
+    "semi": "off",
+    "indent": "off",
+    "comma-dangle": "off",
+    "unicorn/consistent-function-scoping": "off",
+    "max-params": "off",
+    "no-return-await":"off",
+    "no-process-exit":"off",
+    "no-trailing-spaces":"off",
+    "unicorn/no-process-exit":"off"
```

Comment:
Why did we disable this many lint checks.?

Reply:
@antonyagustine reference from contentstack/cli/packages/contentstack-branches/.eslintrc